### PR TITLE
ADD: Show alert on Android if storage access is denied.

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/r9p56kpgfzpeu4tn0y7ytafr5c

[download apk](http://filestorage.bluewallet.io:1488/1606140905.apk)

File: blue_modules/fs.js [hunk 1]
```diff
@@ -39,6 +39,16 @@ const writeFileAndExport = async function (filename, contents) {
       alert(loc.formatString(loc._.file_saved, { filePath: filename }));
     } else {
       console.log('Storage Permission: Denied');
+      Alert.alert(loc.send.permission_storage_title, loc.send.permission_storage_denied_message, [
```
Comment (Overtorment) [inline blue_modules/fs.js:42]: this is generalized function to write to file. dont mention the transaction, only something like "we dont have enough permissions to save file". 
and use this generalized text everywhere else in this pr

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6tbfd0tq871zz1u0zakyak778c

[download apk](http://filestorage.bluewallet.io:1488/1606142666.apk)

**Review by ncoelho — APPROVED** [review]

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

File: screen/send/psbtWithHardwareWallet.js [hunk 2]
```diff
@@ -178,7 +180,7 @@ const PsbtWithHardwareWallet = () => {
       await RNFS.writeFile(filePath, typeof psbt === 'string' ? psbt : psbt.toBase64());
       Share.open({
         url: 'file://' + filePath,
-        saveToFiles: isDesktop,
```
[the remark this answers is not in the document]
Comment (marcosrdz) [reply screen/send/psbtWithHardwareWallet.js:181]: isDesktop is needed to determine if its catalina or big sur. if its catalina, use launchCamera, if its big sur use ScanQRcode

**Pull request merged**